tests/conftest.py

```diff
@@ -56,3 +64,45 @@ def init_db(db_file):
 def db():
     with tempfile.TemporaryDirectory() as tmpdirname:
         yield init_db("{}/vulnerability.db".format(tmpdirname))
+
+
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
Unit tests for different components should be placed to their own subdirectories under `tests/` so that every component can have it's own `conftest.py` with the fixtures and setup it needs. E.g. this fixture has `autouse`, but it's not needed for e.g. manager unit tests.
You can move this unit tests to `tests/evaluator`, moving of manager unit tests can happen in separate PR.

Reply:
moved 51405119260a0e7c06ca1734f368b84bb6f6824d